setup.py

```diff
@@ -74,7 +74,9 @@ def generate_proto(source):
     python_requires='>=3.9',
     install_requires=[
         'ml_dtypes>=0.2.0',
-        'numpy>=1.22',
+        'numpy>1.22',
```

Comment:
What's the reason for doing this?

Won't the current condition satisfy every python version?

Reply:
That's a typo 😅 . Fixed!

Reply:
Ohh sorry, I meant this change. Why do we need this?

Reply:
Numpy will try to build itself when wheels are not available and you are likely to get errors, this happens for older numpy versions on newer python versions. More concretely this was crashing Flax's CI, we added a fix there. I ported it here in case is useful for other projects.

Reply:
Won't pip install the latest available numpy version for the available python version? I think it does that right?

Reply:
Numpy itself doesn't have a constraint on the python version in case you actually want to try to setup everything to build it.